dashboard/pom.xml

```diff
@@ -204,11 +204,11 @@
                                 </goals>
                                 <configuration>
                                     <target>
-                                      <!-- install dependencies -->
-                                      <exec dir="${basedir}" executable="yarn" failonerror="true">
-                                        <arg value="install" />
-                                        <arg value="--ignore-optional" />
-                                      </exec>
+                                        <!-- install dependencies -->
+                                        <exec dir="${basedir}" executable="yarn" failonerror="true">
```

Comment:
revert this change?

Reply:
It's what is done by `mvn fmt:format`, so I would leave it as is to avoid reverting each time after executing of `mvn fmt:format`.